src/main/kotlin/com/demonwav/mcdev/i18n/I18nFoldingSettings.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+
+@State(name = "I18nFoldingSettings", storages = arrayOf(Storage("minecraft_dev.xml")))
```

Comment:
I store the folding settings for Mixin directly in the main config (`editor.codeinsight.xml`), see https://github.com/minecraft-dev/MinecraftDev/pull/238/files#diff-88d71cb2c37a8e8cb611708b891252afR18

I'm not sure which way is better but we should decide for one and use it consistently. (I guess I assumed it would be necessary to store it in that file so we could probably just change the Mixin settings to `minecraft_dev.xml`)